features/projects/components/project-list/project-list.tsx

```diff
@@ -20,15 +22,17 @@ const List = styled.ul`
 `;
 
 export function ProjectList() {
-  const { data, isLoading, isError, error } = useProjects();
+  const { data, isLoading, isError, error, refetch } = useProjects();
+
+  const handleRefresh = () => refetch();
```

Comment:
Minor nitpick: I don't think you need to assign a const here but you could pass `() => refetch()` directly to the `ErrorMessage` component